src/app/workspace/redux-entities/workspace-reducer.ts

```diff
@@ -0,0 +1,39 @@
+import { WorkspaceActions } from './workspace-actions';
+import { FluxStandardAction as Action } from 'flux-standard-action';
+
+interface IState {}
+interface IWorkspace {}
+
+const defaultWorkspaceState = { items:  [] as IWorkspace[], loading: false, error: false };
+
+export default function workspacesReducer(state: IState = defaultWorkspaceState, action: Action<string, IWorkspace>) {
+    switch (action.type) {
+        case WorkspaceActions.LOAD_WORKSPACES_STARTED: {
+            return { ...state, ...{ loading: true } };
```

Comment:
Rewrite everything according to the following scheme
`{ ...state, ...{ loading: true } }` become `{ ...state, loading: true }`